Sources/TuistCoreTesting/Cache/MockCacheDirectoriesProvider.swift

```diff
@@ -6,13 +6,29 @@ import TuistSupport
 public final class MockCacheDirectoriesProvider: CacheDirectoriesProviding {
     private let directory: TemporaryDirectory
     public var cacheDirectoryStub: AbsolutePath?
+    public var currentDirectoryStub: AbsolutePath?
 
     private var cacheDirectory: AbsolutePath {
         cacheDirectoryStub ?? directory.path.appending(component: "Cache")
     }
 
     public func cacheDirectory(for category: CacheCategory) -> AbsolutePath {
```

Comment:
If we have `CacheCategory.directoryName` we can simplify this method and remove some duplicities with the `CacheDirectoriesProvider.swift` file.